README.md

```diff
@@ -16,9 +16,31 @@ curl -sSL https://install.rocketgraph.com/install.sh | sh
 powershell -ep Bypass -c "iex (iwr -useb 'https://install.rocketgraph.com/install.ps1')"
 ```
 
+### Opening a Terminal/Shell
+
+Before running the installation commands, you'll need to open a terminal or command prompt:
+
+Windows:
+
+ - PowerShell: Right-click on the Start button and select "Windows PowerShell" or "Windows Terminal" (on newer Windows versions)
```

Comment:
Change to: Right-click on the Start button and select  "Terminal"

That's all I see on Windows 11.  Win + X does the same thing so change that line also.